cookbooks/bcpc-hadoop/recipes/namenode_no_HA.rb

```diff
@@ -122,31 +138,31 @@
 # We'd prefer to do it after all nodes are members of the HDFS system
 #
 bash "create-hdfs-temp" do
```

Comment:
It would be good to pull out the common code from `namenode_master.rb` and `namenode_no_HA.rb` into a separate recipe and include that recipe here.